src/components/product-filters/product-filters.hbs

```diff
@@ -89,13 +89,18 @@
             <form data-js="price-filter" class="price-filter__form accordion__panel filter__options">
                 <input type="hidden" name="priceFilterFrom" id="priceFilterFrom" value="">
                 <input type="hidden" name="priceFilterTo" id="priceFilterTo" value="">
+                <p class="error-summary hidden" id="errorSummaryPriceFilters" role="alert">You have <span></span> in your price range.</p>
                 <fieldset class="price-filter__fieldset">
                     <legend id="priceLegend" class="visibly-hidden">Filter by Price</legend>
                     <label for="priceFrom" class="price-filter__label">
-                        <span class="sr-only">Price </span>From <input type="text" name="priceFrom" id="priceFrom" class="price-filter__input">
+                        <span class="sr-only">Price </span>From
+                        <span class="error-msg hidden" id="priceFromErrorMsg">Enter a number between 0 and 999.</span>
```

Comment:
@liskulic Because the error is within the label, when the error is visible, it is being read out as part of the label, and then again as the aria-describedby. I think it should be below the label (I tried this quickly on my local, but it seems that the JS targeting relies on it being within the label.)

Reply:
@brianelton, updated.